testing/simple/src/main/scala/org/example/Foo.scala

```diff
@@ -18,15 +18,10 @@ object Foo extends App {
 
   val fn: String => Int = str => str.size
 
-  println(fn.apply("foobar"))
-  println(Baz.fn("foobar"))
+  fn("foobar")
 }
 
 // for SearchServiceSpec
 case class CaseClassWithCamelCaseName()
 case class Bloo()
 case object Blue
-
-object Baz {
-  val fn: String => Int = str => str.size
```

Comment:
This was replaced by [this](https://github.com/ensime/ensime-server/blob/bc4ad93e2fc533f11ead04a66c6d7d476862dd5b/testing/simple/src/main/scala/org/example/Foo.scala#L19-L21).